### DISCUSSION TITLE
GPX recording when device "asleep"

### DISCUSSION THREAD

**Discussion by joffhk** [OpenOrienteering/mapper#2085] [Questions & Answers]
Hi, I'm new to OOM so this might just be asking for help. I want to be able to keep a record of where I was for the WHOLE of the surveying time, not just when the tablet/phone is "awake". It seems that when the device is "asleep" GPX recording is paused, and a straight line just connects recordings when the device is awake. Is there some way to record the whole track? 
PS I have selected the OOM app to "run in background" when the device is asleep

**Comment by pkturner** [comment]
I don't recall how much this was a problem for me when I started using Mapper on my tablet. Now I have the tablet set so that it does not go to sleep on its own. I have the habit of clicking the "power button" to lock the screen whenever I put the tablet in my pocket. (I love that my pants have a pocket just big enough for the tablet.) With this setup, Mapper records complete GPX tracks.

**Comment by joffhk** [comment]
Hi Scott

many thanks for posting reply.

In fact, I have been doing exactly the same as you .. except perhaps that I’m connecting to an accurate GNSS (UBX-F9) via the Bluetooth GNSS app. I was, like you, turning the phone “off” when putting in my pocket, and waking up when ready … for sure, the GPS tracks were NOT recorded while the phone was asleep, only when the phone was awake, but I’ve just done another test outside and it’s now recording the whole track …?!

What GNSS device are you using … internal or BT?

… so let me do a few more tests, with Bluetooth GNSS and the GPS in the phone. You never know with computer devices … is it the machine or the human at fault?! : )

BRgds. Geoff

On 6 Sep 2022, at 23:30, Scott Turner ***@***.***> wrote:


I don't recall how much this was a problem for me when I started using Mapper on my tablet. Now I have the tablet set so that it does not go to sleep on its own. I have the habit of clicking the "power button" to lock the screen whenever I put the tablet in my pocket. (I love that my pants have a pocket just big enough for the tablet.) With this setup, Mapper records complete GPX tracks.

**Reply by pkturner** [reply]
I use a Garmin GLO 2 external device with the Bluetooth GPS app.

**Comment by joffhk** [comment]
Thanks, Scott

I too had a Garmin GLO and it worked well, but now have a U-blox F9 multi-band receiver with external aerial. It gets +/- 1m accuracy at all times, and when connected to RTK via mobile data can get down to 10cm. Would highly recommend for any detailed bush maps … I can mark the south side of a boulder, then mark the north side and the difference in position is often exactly the width of the boulder! : )

I think what happened was that I was testing OOM Mapper without changing it to “remain running when asleep” in the settings. I then remembered and changed the settings to keep running, and tested again … but without closing OOM and restarting … so the settings didn’t take effect I guess.

Now it works perfectly : )

But two things disappoint me - can’t make the GPX track a bit thicker, and really annoying is that the little blue present location pins are NOT saved in the GPX file. My old GPX tracking app saved them, including the option to give them a name etc.

Oh well … thanks again for your advice, much appreciated. Happy mapping, Geoff

On 8 Sep 2022, at 10:26, Scott Turner ***@***.***> wrote:


I use a Garmin GLO 2 external device with the Bluetooth GPS app.
